DotNet/DataAcquisition/Controllers/AuthenticationConfigController.cs

```diff
@@ -98,20 +100,25 @@ private async Task SendAudit(string message, string correlationId, string facili
 
             return Ok(result);
         }
-        catch (BadRequestException bex)
+        catch (BadRequestException ex)
         {
-            await SendAudit(bex.Message, null, facilityId, AuditEventType.Query, null); _logger.LogWarning(bex.Message);
-            return BadRequest(bex.Message);
+            await SendAudit(ex.Message, null, facilityId, AuditEventType.Create, null); _logger.LogWarning(ex.Message);
+            return Problem(title: "Bad Request", detail: ex.Message, statusCode: (int)HttpStatusCode.BadRequest);
         }
-        catch (NotFoundException nex)
+        catch (NotFoundException ex)
         {
-            await SendAudit(nex.Message, null, facilityId, AuditEventType.Query, null);
-            return NotFound(nex.Message);
+            await SendAudit(ex.Message, null, facilityId, AuditEventType.Create, null); _logger.LogWarning(ex.Message);
```

Comment:
Pretty sure we don't need these sendAudit messages anymore as our monitoring instrumentation has been improved. @amphillipsLGC can you confirm?